Userland/Libraries/LibJS/Bytecode/Op.cpp

```diff
@@ -698,6 +699,54 @@ ThrowCompletionOr<void> GetNewTarget::execute_impl(Bytecode::Interpreter& interp
     return {};
 }
 
+ThrowCompletionOr<void> GetImportMeta::execute_impl(Bytecode::Interpreter& interpreter) const
```

Comment:
This looks like a copy of `MetaProperty::execute` from the AST interpreter. Any chance you could move that instead to a helper function that takes ` VM&` and can be called from AST and bytecode?